sydent/util/emailutils.py

```diff
@@ -73,7 +74,10 @@ def sendEmail(
     # conflicting with it.
     allSubstitutions["multipart_boundary"] = generateAlphanumericTokenOfLength(32)
 
-    mailString = open(templateFile).read() % allSubstitutions
```

Comment:
were we not keeping both systems around for the time being?
(perhaps I've missed the memo)

Otherwise it would be nice to check whether the `j2` version exists first, use that if present, otherwise fall back to old-school substitution?